apps/ledger-live-desktop/src/renderer/modals/Send/steps/Confirmation/NodeError/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { Flex, Icons, Text } from "@ledgerhq/react-ui";
+import styled, { useTheme } from "styled-components";
+import { CircleWrapper } from "~/renderer/components/CryptoCurrencyIcon";
+import { useExportLogs } from "LLD/hooks/useExportLogs";
+import { createSendFlowError } from "@ledgerhq/live-common/errors/sendFlow";
+import { urls } from "~/config/urls";
+import { openURL } from "~/renderer/linking";
+import { useLocalizedUrl } from "~/renderer/hooks/useLocalizedUrls";
+import { useCopyToClipboard } from "~/newArch/hooks/useCopyToClipboard";
+import { useTranslation } from "react-i18next";
+import TechnicalErrorSection from "./TechnicalErrorSection";
+import HelpSection from "./HelpSection";
+
+type Props = {
+  error: Error;
+  currencyName: string;
+  networkName: string;
+};
+
+const InteractIcon = styled(Flex)`
+  &:hover {
+    cursor: pointer;
+  }
+  column-gap: 8px;
+`;
+
+const NodeError = ({ currencyName, networkName, error }: Props) => {
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const { handleExportLogs } = useExportLogs();
+  const { copyToClipboard } = useCopyToClipboard();
+
+  const customError = createSendFlowError(error);
```

Comment:
Hey I'm still struggling with the UI on LLM but I saw that the LLM codebase relies on the error names to show the error messages straight from the translation file. It's a bit verbose but I think it's clever as it makes the messages easy to update. I'm not sure if it's the same in LLD but just in case I pushed a draft: [#8081](https://github.com/LedgerHQ/ledger-live/pull/8081/files#diff-86e7954ab3d823c424cf12e3320ccabf3be653f81a8d4ee89c3163d6ac9ddc78).

Basically I added a `UnknownSendFlowError` returned by default by [sendFlow.ts](https://github.com/LedgerHQ/ledger-live/pull/8081/files#diff-44f82b6515fd35cd9afdfb74b6a3139974c9c9ed27524e0b3e71528829a3af6c) then by passing this error to `GenericErrorView` and adding the translations it just show the correct error.

Reply:
There is the same on LLD but I'm not sure that it would be worth to add since we don't have specific message to display for each error. But we can discuss of it with product eventually if they want to add custom message in the future it will definitely be worth